packages/account-tree-controller/src/AccountTreeController.ts

```diff
@@ -501,6 +506,11 @@ export class AccountTreeController extends BaseController<
         }
       });
 
+      this.messagingSystem.publish(
+        `${controllerName}:accountTreeUpdated`,
```

Comment:
I also wonder if we should go for `:accountTreeChange` to re-use the same naming convention of `:stateChange` event 🤔 We already have some `*Updated` event in some other controllers, but they might not always be related to their state, while the `accountTree` here is one of the main element of this controller's state 😅 WDYT?